src/App.js

```diff
@@ -15,6 +17,9 @@ function App() {
           <Route path="timerpage" element={<TimerPage />} />
           <Route path="settings" element={<Settings />} />
           <Route path="knowmore" element={<KnowMore />} />
+          <Route path="login" element={<Login />} />
+          <Route path="signup" element={<Signup />} />
```

Comment:
Covering the first 3 routes in a Private Route component  (meant only for logged in users ) related to TimerPage,Settings and KnowMore would be better!!

Reply:
You're right! Thank you for this suggestion, will surely work on it.